dsbulk/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Copyright DataStax, Inc, 2017
+#   Please review the included LICENSE file for more information.
+#
+
+FROM dse-base
+
+MAINTAINER "DataStax, Inc <info@datastax.com>"
+
+ARG VERSION=1.3.3
+ARG URL_PREFIX=http://localhost
+ARG DSBULK_TARBALL=dsbulk-${VERSION}.tar.gz
+ARG DSBULK_DOWNLOAD_URL=${URL_PREFIX}/${DSBULK_TARBALL}
+
+
+ENV DSBULK_HOME /dsbulk
+
+RUN set -x \
+# Add cassandra user
+    && groupadd -r cassandra --gid=999 \
+    && useradd -m -d "$DSBULK_HOME" -r -g cassandra --uid=999 cassandra
+
+# Install missing packages
+RUN set -x \
+    && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
does dsbulk really need these dependencies? by looking at the official [docs](https://docs.datastax.com/en/install/6.7/install/installDSbulk.html), there is nothing about it.

Reply:
I'll look, I need to test it more...